applications/deployments/osd-config-custom-domains-applications.adoc

```diff
@@ -0,0 +1,8 @@
+[id="osd-config-custom-domains-applications"]
+= Configuring custom domains for applications
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: osd-config-custom-domains-applications
+
+toc::[]
+
+include::modules/osd-applications-config-custom-domains.adoc[leveloffset=+2]
```

Comment:
The leveloffset=+2 makes this render as an H3 in your preview. Is that intentional?